Prace_domowe/Praca_domowa3/Grupa4/Mateusz_Stączek/notebook.ipynb

```diff
@@ -0,0 +1,239 @@
+{
```

Comment:
<p>KNN też ma chyba overfitting, ogólnie to spoko</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Jakiś komentarz, która miara jest ważna też mile widziany</p>